api/routes.py

```diff
@@ -11,4 +16,9 @@
         CityCouncilAgendaView.as_view(),
         name="city-council-agenda",
     ),
+    path(
+        "city-council-attendance-list/",
```

Comment:
Estava pensando em separarmos a rota por órgão:

```suggestion
        "city-council/attendance-list/",
```

Você poderia fazer o mesmo refactoring em `city-council-agenda`, porfa?

Reply:
Sim!